.editorconfig

```diff
@@ -6,3 +6,7 @@ indent_size = 4
 trim_trailing_whitespace = true
 end_of_line = lf
 insert_final_newline = true
+
+# yarn / npm uses 2 spaces when it dumps the file after a change
+[package.json]
```

Comment:
Given that .eslintrc is a json file, should we include it here too for consistency?

Reply:
The rest of the codebase is 4 spaces!
The problem is that when you do yarn / npm install with old versions, they'll reformat the file to 2 spaces.

This is just to try to stop future vscode autoformat on save problems.

Reply:
Ah, that makes sense... let's forget about this